scripts/generate_maps.py

```diff
@@ -0,0 +1,202 @@
+
+def plot_data_on_basemap(basemap, gdf, institution, filename, output_folder):
+    fig, ax = plt.subplots(figsize=(10, 10))
+
+    # Set background color to white
+    fig.patch.set_facecolor('white')
+    ax.set_facecolor('white')
+
+    # Plot basemap with custom colors
+    basemap.plot(ax=ax, color=basemap['color'], edgecolor='black')
+
+    # Define colors and labels based on availability
+    availability_colors = {'u': '#fb9a99', 's': '#1f78bc', 'o': 'grey'}
+    availability_labels = {'u': 'Unavailable', 's': 'Available', 'o': 'Outdated'}
+
+    # Plot the data for the institution
+    if not gdf.empty:
+        for availability in ['s', 'u', 'o']:  # Ensure the order of plotting
+            subset = gdf[gdf['availability'] == availability]
+            if not subset.empty:
+                color = availability_colors.get(availability, 'darkgrey')
+                label = availability_labels.get(availability, 'Other')
+                subset.plot(ax=ax, color=color, markersize=0.55, linewidth=0.25, label=label)
+
+    # Set limits to zoom in on Antarctica
+    ax.set_xlim(-3e6, 3e6)
+    ax.set_ylim(-3e6, 3e6)
+
+    # Set equal aspect ratio for circular plot and remove axes
+    ax.set_aspect('equal')
+    ax.axis('off')
+
+    # Add custom scale bar with "1000 km" label only
+    scalebar = ScaleBar(1, location='lower right', units='km', dimension='si-length', label='1000 km', length_fraction=0.2)
+    scalebar.dx = 1  # Set the distance to 1 unit of whatever is specified (km in this case)
+    scalebar.label = '1000 km'  # Custom label
+
+    ax.add_artist(scalebar)
+
+    plt.title(f'{institution} Data Availability', fontsize=14)
+
+    # Handle legend
+    legend_patches = [Patch(color=color, label=availability_labels[availability]) for availability, color in availability_colors.items()]
+    ax.legend(handles=legend_patches, loc='upper right', fontsize=8, title='Availability')
+
+    # Draw a circular boundary
+    circle = plt.Circle((0, 0), 3e6, transform=ax.transData, color='black', fill=False, linewidth=1)
+    ax.add_artist(circle)
+
+    # Save the plot to the data folder
+    output_path = os.path.join(output_folder, filename)
+    plt.savefig(output_path, bbox_inches='tight', pad_inches=0.1)
+    plt.close(fig)
+    print(f"Saved map for {institution} to {output_path}")
+
+def main(input_file_path, map_path, output_folder, statistics_folder):
+    # Ensure the output folders exist
+    os.makedirs(output_folder, exist_ok=True)
+    os.makedirs(statistics_folder, exist_ok=True)
+
+    # Read the GeoPackage file into a GeoDataFrame
+    fp = gpd.read_file(input_file_path)
+
+    # Read the map shapefile
+    mp = gpd.read_file(map_path)
+
+    # Check if the 'Category' column exists
+    if 'Category' not in mp.columns:
+        print("The 'Category' column does not exist in the GeoDataFrame.")
+        return
+
+    # Define the colors for the categories
+    category_colors = {
+        'Ocean': '#a3bdd1',
```

Comment:
Ah-hah! I think this is where you can turn your square into a circle ... I'd try making the ocean transparent, then draw a circle with this as the facecolor.